source/Camera/Basic/CaptureHDRCompleteSettings/CaptureHDRCompleteSettings.cpp

```diff
@@ -20,8 +45,9 @@ int main()
         auto camera = zivid.connectCamera();
 
         std::cout << "Configuring global processing settings:" << std::endl;
```

Comment:
This comment no longer makes sense, since you configure all settings at once here.

Reply:
Reverted to old method, this now makes sense if we want to keep it this way